### PR TITLE
nordicbits: fix title, imdb search and dead torrents. resolves #8178

### LINKED ISSUE #8178: NordicBits Issues: Extra leading space | IMDb ID search implementation | Search discrepancies
- [ ] The search results returned from Jackett has an added "space" / empty character before the actual search result. This sometimes confuses further processing of the result. 

Here's an example...

image

***
@RoloSoze Edit:

I was actually going to open a new ticket for NordicBits just now, but since you opened one, I'll add a couple more issues:

- [ ] Searching with the IMDb ID in NordicBits is the same as doing an empty string search - It returns the last 100 torrents. It needs to be configured correctly - https://github.com/Jackett/Jackett/issues/4859#issuecomment-613002060

- [ ] I tried searching for 'Avengers' in Jackett - Got 36 results (including some dead torrents). Searching NordicBits' website using the same word returns 40 results. It appears that Jackett returns some but not all dead torrents, probably depending on the category. However, I'm not entirely sure that this is the only issue, but that's what I found through a series of tests.

### PR DESCRIPTION
**Pull request by ngosang**
Tested

### CODE REVIEW

**Comment by ngosang** [comment]
@RoloSoze

File: src/Jackett.Common/Indexers/Nordicbits.cs [hunk 1]
```diff
@@ -462,30 +462,22 @@ protected override async Task<IEnumerable<ReleaseInfo>> PerformQuery(TorznabQuer
         /// <param name="url">Search url for provider</param>
         /// <param name="page">Page number to request</param>
         /// <returns>URL to query for parsing and processing results</returns>
-        private string BuildQuery(string term, TorznabQuery query, string url, int page = 0)
+        private string BuildQuery(string searchTerm, TorznabQuery query, string url, int page = 0)
         {
-            var parameters = new NameValueCollection();
+            var qc = new NameValueCollection();
```
Comment (cadatoiva) [inline src/Jackett.Common/Indexers/Nordicbits.cs:467]: move lower line into object initializer.
```suggestion
            var qc = new NameValueCollection
            {
                {"incldead", "1"}
            };
```

File: src/Jackett.Common/Indexers/Nordicbits.cs [hunk 2]
```diff
@@ -523,15 +515,10 @@ private string BuildQuery(string term, TorznabQuery query, string url, int page
...
-            var CatQryStr = "";
-            foreach (var cat in categoriesList)
-                CatQryStr += cat + "&";
-
             // Building our query
-            url += "?" + CatQryStr + searchterm + "&" + parameters.GetQueryString();
+            url += "?" + qc.GetQueryString() + "&" + string.Join("&", categoriesList);
 
-            Output("\nBuilded query for \"" + term + "\"... " + url);
+            Output("\nBuilded query for \"" + searchTerm + "\"... " + url);
```
Comment (cadatoiva) [inline src/Jackett.Common/Indexers/Nordicbits.cs:521]:
```suggestion
            Output("\nBuilt query for \"" + searchTerm + "\"... " + url);
```

File: src/Jackett.Common/Indexers/Nordicbits.cs (hunk 1 above)
Comment (ngosang) [reply src/Jackett.Common/Indexers/Nordicbits.cs]: fixed

**Pull request merged**